LayoutTests/platform/ios/TestExpectations

```diff
@@ -7523,6 +7523,9 @@ imported/w3c/web-platform-tests/mathml/presentation-markup/operators/embellished
 imported/w3c/web-platform-tests/mathml/relations/html5-tree/tabindex-002.tentative.html [ Skip ]
 imported/w3c/web-platform-tests/mathml/relations/css-styling/mozilla-393760-1.xml [ ImageOnlyFailure ]
 
+webkit.org/b/214290 imported/w3c/web-platform-tests/css/css-text/text-transform/math/text-transform-math-auto-001.html [ ImageOnlyFailure ]
```

Comment:
as I mentioned in the other PR, I assume you want to force the pref to be enabled for the tests in `imported/w3c/web-platform-tests/css/css-text/text-transform/math/` so result is more platform independent.